### PR TITLE
[mod_languages]XML code style

### PR DESCRIPTION
**Pull request by wojsmol**
### Summary of Changes

Code style changes for mod_languages
### Testing Instructions

code review

cc @JoomliC @andrepereiradasilva

### CODE REVIEW

**Comment by andrepereiradasilva** [comment]
I have tested this item :white_check_mark: successfully on 1f088187b60c78a844dcf5056d33cf4aa35e3a7c<br /><br />on code review<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10585">issues.joomla.org/joomla-cms/10585</a>.</sub>

**Comment by brianteeman** [comment]
I have tested this item :white_check_mark: successfully on 1f088187b60c78a844dcf5056d33cf4aa35e3a7c<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10585">issues.joomla.org/joomla-cms/10585</a>.</sub>

**Comment by brianteeman** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10585">issues.joomla.org/joomla-cms/10585</a>.</sub>

File: modules/mod_languages/mod_languages.xml [hunk 1]
```diff
@@ -34,114 +35,117 @@
...
 					<option value="0">JNO</option>
 				</field>
 
 				<field
 					name="full_name"
 					type="radio"
+					label="MOD_LANGUAGES_FIELD_FULL_NAME_LABEL"
+					description="MOD_LANGUAGES_FIELD_FULL_NAME_DESC"
 					class="btn-group btn-group-yesno"
 					showon="dropdown:0[AND]image:0"
```
Comment (JoomliC) [inline modules/mod_languages/mod_languages.xml]: No rule here (so valid), but i prefer to have <code>showon</code> attribute always at end... 
What's your opinion here ? cc/ @wojsmol @andrepereiradasilva

Comment (andrepereiradasilva) [inline modules/mod_languages/mod_languages.xml]: since there is no rule for that. this way is ok

**Pull request closed without merging**